src/Tools/dotnet-monitor/CollectionRules/Options/Actions/CollectTraceOptions.cs

```diff
@@ -63,5 +63,10 @@ internal sealed partial record class CollectTraceOptions : BaseRecordOptions, IE
         [ValidateEgressProvider]
```

Comment:
Eventually we'll need to change the max here, but I don't think it has to be part of this work.